central/views/imagecveflat/singleton.go

```diff
@@ -0,0 +1,36 @@
+package imagecveflat
+
+import (
+	"github.com/stackrox/rox/central/globaldb"
+	"github.com/stackrox/rox/pkg/features"
+	"github.com/stackrox/rox/pkg/postgres"
+	"github.com/stackrox/rox/pkg/postgres/schema"
+	"github.com/stackrox/rox/pkg/sync"
+)
+
+var (
+	once sync.Once
+
+	imageCVEFlatView CveFlatView
+)
+
+// NewCVEFlatView returns the interface CveView
+// that provides searching image cves stored in the database.
+func NewCVEFlatView(db postgres.DB) CveFlatView {
+	if !features.FlattenCVEData.Enabled() {
+		return nil
+	}
+
+	return &imageCVEFlatViewImpl{
+		db:     db,
+		schema: schema.ImageComponentV2Schema,
```

Comment:
Why run the query on componentV2 schema and not CVEV2 schema?

Reply:
That's a nice catch.  Component schema works because CVE references component.  But CVEV2 schema would be more correct.